docker/enterprise/Readme.md

```diff
@@ -0,0 +1,65 @@
+# Novu Enterprise: On-Premises Non-Production Trial with Docker Compose
+
+This documentation guides you through the process of setting up an on-premises trial of Novu Enterprise using Docker Compose.
+
+To do this trail you will need to contact [sales](https://notify.novu.co/meetings/ryannovu/30minutes) to get a token.
+
+
+
+Note: This is not our preferred way of doing an enterprise trail,
+all features are available on Novu Cloud with a free trial.
+
+## Before You Begin
+
+Ensure that the following software has been installed on your system:
+
+- [Docker](https://docs.docker.com/engine/install/) and [docker-compose](https://docs.docker.com/compose/install/)
+- [Git](https://git-scm.com/downloads)
+
+## Guide
+
+### Get the Code
+
+Warning: Before you start this process, If you have an existing mongodb volume you will need to remove it to 
+ensure the enterprise tables are installed correctly.
+
+Clone the Novu repository and navigate to the Docker directory:
+
+```sh
+# Add the token to the shell env
+GH_TOKEN=ghp_xxxx
+
+# Get the code
+git clone https://github.com/novuhq/novu
+
+# Go to the docker folder
+cd novu/docker
+
+# Copy the example env file
+cp .env.example ./enterprise/.env
+
+# Login
+echo "$GH_TOKEN" | docker login --username cliftonz --password-stdin
```

Comment:
exchange specific username with a generic one

Reply:
I cant the token is specifically tied to my account, but I can include that in the sales email instead of publicly.

Thanks!